src/uwtools/tests/config/test_tools.py

```diff
@@ -645,12 +645,15 @@ def test__realize_config_values_needed_negative_results(caplog, tmp_path):
 @pytest.mark.parametrize("output_fmt", FORMAT.extensions())
 def test__validate_format_output(input_fmt, output_fmt):
     call = lambda: tools._validate_format_output(input_fmt=input_fmt, output_fmt=output_fmt)
-    if input_fmt in (FORMAT.yaml, output_fmt):
+    if FORMAT.yaml in (input_fmt, output_fmt) or input_fmt == output_fmt:
```

Comment:
Comments offline note that this does not ensure that the actual rendering is correct or expected. Particularly, namelist dicts are understood by YAML with !! notation but are not rendered as expected.

Reply:
This issue is now covered by the additional YAML representer